composer.json

```diff
@@ -23,7 +23,7 @@
             "@lint"
         ],
         "lint": [
-            "vendor/bin/yaml-lint *.yml",
+            "for FN in *.yml *.yaml; do vendor/bin/yaml-lint \"$FN\"; done",
```

Comment:
`FN` is quite misleading.. at the first look I thought it's `function` .. why is it not just `fileName`?

Reply:
FN is from my experience one of the most used shortcut in bash scripts... However obviously not enough generally know, I will change it :)